CHANGELOG.md

```diff
@@ -3,6 +3,9 @@
    instead of using Thread.raise on the plugins' threads. Ref: https://github.com/elastic/logstash/pull/3895
  - Dependency on logstash-core update to 2.0
 
+# v 0.1.4
```

Comment:
in v1 branch, changelog should show the fix for 0.1.4 while for v2 the fix goes in for the next available version. makes sense?